README.md

```diff
@@ -53,9 +53,18 @@ php artisan budget:update
 
 ## Docker
 
-You can run Budget using Docker. Spinning it up (`docker-compose up`) will create 3 containers–for MySQL, PHP and NGINX.
-
-*Disclaimer–currently the Docker setup is missing cronjobs and the queue worker.*
+You can run Budget using Docker Compose. Docker starts 2 containers - one for the database and the other for the application itself.
```

Comment:
```diff
- .. using Docker Compose. Docker starts 2 containers ..
+ .. using Docker Compose. Docker Compose starts 2 containers ..
```

Trying to be as specific as possible, to prevent misunderstandings.